docs/UserGuide.md

```diff
@@ -200,6 +200,34 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the contacts.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Marking a person : `mark`
+
+Marks the specified person from the contacts as favourite.
+
+Format: `mark INDEX`
+
+* Marks the person at the specified `INDEX`.
+* The index refers to the index number shown in the displayed person list.
+* The index **must be a positive integer** 1, 2, 3, …​
+
+Examples:
+* `list` followed by `mark 2` marks the 2nd person in the contacts.
+* `find Betsy` followed by `mark 1` marks the 1st person in the results of the `find` command.
+
+### Unmarking a person : `unmark`
```

Comment:
For clarity, maybe it would be better for the header for `Mark` and `Unmark` commands to be
`Marking a person _as 'Favourite'_`?

Reply:
Oh yes, thank you for pointing it out 👍